exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -364,18 +363,47 @@ def test_duplicated_path_mapping_raises_exception(self):
                 p1.mkdir(parents=True)
                 with open(test_file1, "w") as f:
                     f.write("test")
-                p2 = Path(d2) / "level0" / "level1_0" / "test"
-                test_file2 = p2 / "test.txt"
+                p2 = Path(d2) / "level0" / "level1_1" / "test"
                 p2.mkdir(parents=True)
                 test_file2 = p2 / "test.txt"
                 with open(test_file2, "w") as f:
                     f.write("test")
 
                 path1 = p1.parent
                 path2 = p2.parent
-                self.assertRaises(AssertionError, lambda: hasher_content_only.hash([PathMapping("test", str(path1)),
-                                                                                    PathMapping("test", str(path2))]))
+                path_mappings = [PathMapping("test", str(path1)), PathMapping("test", str(path2))]
+                self.assertRaises(AssertionError, lambda: hasher_content_only.hash(path_mappings))
 
+    def test_duplicated_path_mapping_with_subpath_raises_exception(self):
+        """
+        Test that a duplicated mapping raises an exception.
```

Comment:
Now we only miss a test where the destination are subpaths and the sources are constructed to cause a duplicate